_plugins/ref_tag.rb

```diff
@@ -11,7 +11,7 @@ def render(context)
     registers = context.registers
     site = registers[:site]
     node = site.collections['nodes'].docs.detect { |doc| doc.data['slug'] == @slug }
-    "<a href='#{site.baseurl}#{node.url}' class='slug'>[#{@slug}]</a>"
+    "<a href='#{site.baseurl}#{node.url}' role='tooltip' aria-label='#{node.title}' class='slug'>[#{@slug}]</a>"
```

Comment:
Should this be `node.data["title"]` or something? I am getting some deprecation warnings.

Reply:
Yes! `node.title` is good for templates where the page is serialized, but here we're accessing the `Page` instance directly. Fixed.